app/components/Chat/Chat.css

```
@@ -66,3 +66,21 @@
   opacity: 0.7;
   margin-left: 8px;
 }
+
+.listGroupItemName span.emoji {
+  font-size: 1em;
```

```suggestion
  font-size: 1rem;
```

Preffer rem usage over em. Difference is that rem always calculate based on root font size - em takes parent size which can often change.